test/io-tests/configs/expected/load-db-guc.config

```diff
@@ -0,0 +1,27 @@
+db-anon-role = "postgrest_test_anonymous"
+db-channel = "postgrest"
+db-channel-enabled = true
+db-extra-search-path = "public,extensions"
+db-max-rows = 1000
+db-pool = 1
+db-pool-timeout = 100
+db-pre-request = "prereq_func"
+db-prepared-statements = false
+db-root-spec = "openapi_func"
+db-schemas = "test,tenant1,tenant2"
+db-load-guc-config = "true"
+db-tx-end = "commit-allow-override"
+db-uri = "<REPLACED_WITH_DB_URI>"
+jwt-aud = "https://example.org"
+jwt-role-claim-key = ".\"a\".\"role\""
+jwt-secret = "REALLYREALLYREALLYREALLYVERYSAFE"
+jwt-secret-is-base64 = true
+log-level = "info"
+openapi-server-proxy-uri = "https://example.org/api"
+raw-media-types = "application/vnd.pgrst.config"
```

Comment:
```suggestion
raw-media-types = "application/vnd.pgrst.db-config"
```

Reply:
This test allowed me to fix a bug in the settings query because I was only getting a `"application/vnd.db-config"`. The `pgrst.` stripping from the query was also applied to values :sweat_smile: .